build.gradle

```diff
@@ -13,7 +13,7 @@ allprojects {
         ext.version_log4j = '2.5'
         ext.version_jackson = '2.8.8'
         ext.version_retrofit = '2.1.0'
-        ext.version_slackapi = '0.6.0'
+        ext.version_slackapi = '0c58385'
```

Comment:
Is there a release version of the slack API project that has the changes needed?